test/unit/deferred.js

```diff
@@ -888,7 +889,7 @@ QUnit.test( "jQuery.when(thenable) - like Promise.resolve", function( assert ) {
 		}
 	};
 	if ( typeof Symbol === "function" ) {
-		customToStringThen.then[ Symbol.toStringTag ] = "String";
+		//customToStringThen.then[ Symbol.toStringTag ] = "String";
```

Comment:
This change is incorrect, a `then` method that mimics another type must still be used because that's how native Promises behave.

Reply:
Sure, but the unit test awaits a thenable, not an empty string.

Reply:
Right, it needs to await a thenable _in which the `then` method mimics a non-function_.